numpy/distutils/line_endings.py

```diff
@@ -11,19 +11,20 @@ def dos2unix(file):
         print(file, "Directory!")
         return
 
-    data = open(file, "rb").read()
-    if '\0' in data:
-        print(file, "Binary!")
-        return
+    with open(file, "rb") as fp:
+        data = fp.read()
+        if '\0' in data:
+            print(file, "Binary!")
+            return
```

Comment:
No need to indent these lines - only the lines using `fp` need to be within the `with`

Reply:
Done